openchallenges/service_stack.py

```diff
@@ -157,11 +158,19 @@ def __init__(
             protocol=elbv2.ApplicationProtocol.HTTP,
         )
 
+        health_check = elbv2.HealthCheck(path="/", interval=duration.minutes(1))
+        if "apex" in construct_id:
+            health_check = elbv2.HealthCheck(
```

Comment:
It would be more efficient to update the data of the first `health_check` instead of instantiating a new `elbv2.HealthCheck` object.

Reply:
We use this class to deploy multiple services.  The `if` condition is to target the `apex` service only.  

The first definition of `health_check` is the default configuration.  This one is not a new instance, it is an update to the first instance.

Reply:
Based on the code you don't need to instantiate a new object: the `if` statement reassign `health_check` so you may as well update the existing object.

Reply:
done per your suggestions.

Reply:
nope, can't do your suggestion @tschaffter ...```➜ AWS_PROFILE=itsandbox-admin AWS_DEFAULT_REGION=us-east-1 cdk synth
Traceback (most recent call last):
  File "/Users/zaro0508/work-sage/openchallenges/app.py", line 342, in <module>
    apex_service_stack = LoadBalancedServiceStack(
                         ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/zaro0508/work-sage/openchallenges/.venv/lib/python3.12/site-packages/jsii/_runtime.py", line 118, in __call__
    inst = super(JSIIMeta, cast(JSIIMeta, cls)).__call__(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/zaro0508/work-sage/openchallenges/openchallenges/service_stack.py", line 165, in __init__
    health_check.path = "/health"
    ^^^^^^^^^^^^^^^^^
AttributeError: property 'path' of 'HealthCheck' object has no setter

Subprocess exited with error 1
```

Reply:
It's unfortunate that setters are not defined.